book.rb

```diff
@@ -8,7 +8,7 @@ def initialize(title, author)
     @rentals = []
   end
 
-  def add_rental(rental)
-    @rentals << rental
+  def add_rental(date, person)
+    @rentals << Rental.new(date, self, person)
```

Comment:
*[OPTIONAL]*

- Hey, I think you don't really need to push to the @rentals instance variable from here, you'd add it later so that this doesn't confuse you later in other milestones. Also, in Rentals class, you've already pushed into those arrays `books, person` , You could leave it this way:

`Rentals.new(date, self, person)`